### ISSUE TITLE
Data table - field overflow

### ISSUE THREAD

**Issue by smalers** [OpenWaterFoundation/owf-app-dev-ng#63]
Data table fields sometimes overflow into the next field, which looks bad.   See the example below.  Can anything be done?  Also, why does this data table even have the layer and path?  That seems like internal information that should not be visible to the user, unless perhaps in developer mode.

![image](https://user-images.githubusercontent.com/6448274/183962599-9b91285d-5be3-4486-901d-b545efb93564.png)

**Comment by Nightsphere** [comment]
Both `layer` and `path` properties were in the GeoJSON file, and the code just iterates over each and prints them out in the table. I'll do some testing on extra long non-breaking names and should have a fix for the next Common package version.

**Comment by Nightsphere** [comment]
I have added some extra CSS to the table cells to truncate long, non-breakable strings. I briefly looked at what it would take to create resizable columns, and Angular Material does not have the feature built in. There seems to be a way to do it, but it is not a trivial solution. I'll keep it in mind for any future implementations. Here is how the new code will resolve this issue:

![image](https://user-images.githubusercontent.com/22484403/184245674-2253711b-0558-4f90-a35e-1123e8a6d873.png)

Closing this issue.
